packages/server/utils/gameConfig.ts

```diff
@@ -0,0 +1,59 @@
+import Handle from './handle';
+import { ClientId } from '../objects/client';
+
+export interface GameConfig {
+    /**
+     * A unique identifier for this game in the server.
+     */
+    type: string;
+    /**
+     * The number of players supported. If one number is given,
+     * the game needs exact number of players. If two numbers are given, the number of player
+     * needs to be between the two number inclusively.
+     */
+    playerRange: [number] | [number, number];
```

Comment:
I think the way of setting playerRange should have more flexibility.  What if I want to write a game witch happened to require even number of players?